pkg/getter/getter.go

```diff
@@ -172,9 +172,14 @@ func (p Providers) ByScheme(scheme string) (Getter, error) {
 	return nil, errors.Errorf("scheme %q not supported", scheme)
 }
 
+var defaultOptions = []Option{WithTimeout(time.Second * 120)}
```

Comment:
Can we make the duration a const with a comment explaining how the value was chosen, please?

Reply:
Done.